### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by KaterynaEV**
- [DEMO LINK](https://KaterynaEV.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://KaterynaEV.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,52 @@
...
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="#home">
+    <header class="navigation">
+      <div class="container">
+        <div class="nav__logo">
+          <img src="images/logo.png"
+          alt="Moyo logo"
+          class="logo">
```
Comment (Andriy-Shymkiv) [inline src/index.html:20]:
```suggestion
          <img 
             src="images/logo.png"
             alt="Moyo logo"
             class="logo"
          >
```

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,52 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Bricolage+Grotesque&family=Inter:wght@400;700&family=Raleway:ital,wght@0,400;1,200&family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="#home">
+    <header class="navigation">
+      <div class="container">
```
Comment (Andriy-Shymkiv) [inline src/index.html:16]: I don't think that you need this container. you can give same styles for header

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,52 @@
...
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link"
+              data-qa="hover">Laptops & Computers</a>
```
Comment (Andriy-Shymkiv) [inline src/index.html:37]:
```suggestion
              <a 
                href="#" 
                class="nav__link"
                data-qa="hover"
              >
                Laptops & Computers
              </a>
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  display: block;
```
Comment (Andriy-Shymkiv) [inline src/style.css:7]: remove this style, you don't need it

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,77 @@
...
+}
+
+.container {
+  background-color: #fff;
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
```
Comment (Andriy-Shymkiv) [inline src/style.css:15]: give this styles for the header

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,77 @@
...
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  display: block;
+}
+
+.container {
+  background-color: #fff;
```
Comment (Andriy-Shymkiv) [inline src/style.css:11]: it is a default color, please remove

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,77 @@
...
+  align-items: center;
+}
+
+a {
+  text-decoration: none;
+}
+
+nav {
+  display: inline-block;
+}
```
Comment (Andriy-Shymkiv) [inline src/style.css:24]: do not use selectors by tag. read checklist

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,77 @@
...
+
+.nav__logo {
+  align-self: center;
+  height: 40px;
+  justify-content: center;
```
Comment (Andriy-Shymkiv) [inline src/style.css:29]:
```suggestion
```

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,77 @@
...
+
+.nav__link {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  height: 60px;
```
Comment (Andriy-Shymkiv) [inline src/style.css:51]: use padding instead of hardcoding height

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 5 above)
Comment (KaterynaEV) [reply src/style.css:15]: [file:///C:/Users/Kate/projects/layout_moyo-header/backstop_data/html_report/index.html](url). i don't understand what's wrong. could you help?

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
code looks very good now, just fix the case with height please. try `padding: 22px 0`

File: readme.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://KaterynaEV.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://KaterynaEV.github.io/layout_moyo-header/report/html_report/)
```
Comment (Viktor-Kost) [inline readme.md:4]: please check your deployment since links lead to 404 page

File: src/index.html [hunk 11]
```diff
@@ -6,9 +6,58 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="#home">
+    <header class="header">
+      <div class="nav__logo">
```
Comment (Viktor-Kost) [inline src/index.html:16]: this `div` is redundant
```suggestion
```

File: src/style.css [hunk 12]
```diff
@@ -1,3 +1,68 @@
...
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (Viktor-Kost) [inline src/style.css:67]: move repeated colors to varaibles

File: src/style.css [hunk 13]
```diff
@@ -1,3 +1,68 @@
...
+  padding: 0;
+}
+
+.nav__link {
+  font-weight: 500;
+  line-height: 14px;
+  text-transform: uppercase;
+  text-align: center;
+  display: flex;
+  align-items: center;
```
Comment (Viktor-Kost) [inline src/style.css:40]: you can make links just as block

File: src/style.css [hunk 14]
```diff
@@ -1,3 +1,68 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__logo {
+  height: 40px;
+  object-fit: contain;
+  align-self: center;
```
Comment (Viktor-Kost) [inline src/style.css:20]: looks redundant

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Pull request closed without merging**